### DISCUSSION TITLE
change file count check in add_structure.ps1

### DISCUSSION THREAD

**Discussion by macnaughtani** [damienvanrobaeys/Run-in-Sandbox#57] [Ideas]
Hello Damien. Apologies if this is not the right place for this. As our corporate security software deletes psexec as soon as its detected, and your changelog indicates that psexec is not used anymore, I changed the file count check in add_structure.ps1 from 40 to 39 so that script would continue. I've not tested this tool yet but it looks impressive so fingers crossed that psexec is not required. 

Thanks.
Ian.

**Comment by Joly0** [comment]
Hey @macnaughtani i just wanted to let you know, that damien is only sparsely working on Run-in-Sandbox anymore.

I would recommend my fork over at https://github.com/Joly0/Run-in-Sandbox where i work actively on it.

Also psexec is still used, as it is needed for intunewin testing for running that as system user in the sandbox. The note in the changes that its no longer needed was a mistake.
Although i am searching for an alternative way, there hasnt been any pleasing solution to this problem that could replace psexec (as i would like not to use it anymore aswell).

Otherwise i would recommend asking your companies it team if it was possible to exclude psexec in the path that Run-in-Sandbox uses (atleast thats what i have done at my company)

**Reply by macnaughtani** [reply]
Thanks @Joly0 and apologies for the slow response. I did find your fork and have used that. I am one of the sysadmins but our MSP manages the security software. Asking them to exclude psexec in a specific path is a good idea so I'll give that a try. Thanks again.
